### PR TITLE
fix(microsite): make the anchor link icon show up

### LINKED ISSUE #3707: microsite: make the anchor link icon show up
## Expected Behavior

When hovering over headings on the microsite (https://backstage.io), there should appear a little anchor link to the left. This is how it looks in GitHub when rendering the same page:

Screenshot 2020-12-14 at 13 19 52

## Current Behavior

Screenshot 2020-12-14 at 13 18 41

## Possible Solution

CSS Wizardry of level 3 or higher.

## Steps to Reproduce

1. Go to https://backstage.io/docs/overview/what-is-backstage
2. Hover over any heading, for example Benefits
3. Notice that there's no link icon on the left side, visually, but that it actually exists and is clickable etc (but black so it can't be seen on top of the background)

### PR DESCRIPTION
**Pull request by AdnCodez**
## Hey, I just made a Pull Request!

Closes #3707

Added some CSS in main.css, to fix the color of the anchor on hover effect of each H2.
`h2:hover .hash-link{
  opacity: 1
}

.hash-link {
  fill:white;
}`

#### :heavy_check_mark: Checklist

scst1
scst2

### CODE REVIEW

File: microsite/static/css/custom.css [hunk 1]
```diff
@@ -48,6 +48,14 @@ h6 {
   color: $textColor;
 }
 
+h2:hover .hash-link{
```
Comment (adamdmharvey) [inline microsite/static/css/custom.css:51]: As I noted in #3707 , once you run `yarn prettier --write *`, this file will be re-written, an dthen you need to then re-commit, and push that change up into this AdnDozez:master branch.

Alternatively, the easy fix may just be this...

```suggestion
h2:hover .hash-link {
```

**Comment by freben** [comment]
@AdnCodez You don't need to close these. You can just modify your local branch correctly and then force-push to replace the remote one.

**Comment by AdnCodez** [comment]
oh sorry I didn't seem to have it work directly, so I retried in new one :(

**Comment by OrkoHunter** [comment]
@AdnCodez Jump over on our Discord if you need added #support with any git commands. We are happy to help.

**Comment by AdnCodez** [comment]
Thank you I will, I need to get familiar with the correct workflow :+1:

**Pull request closed without merging**